tardis/montecarlo/montecarlo_numba/r_packet.py

```diff
@@ -341,16 +339,15 @@ def trace_packet(r_packet, numba_model, numba_plasma, estimators):
     """
     Traces the RPacket through the ejecta and stops when an interaction happens (heart of the calculation)
 
-    r_packet: RPacket
     Parameters
     ----------
-    numba_model: tardis.montecarlo.montecarlo_numba.numba_interface.NumbaModel
-    numba_plasma: tardis.montecarlo.montecarlo_numba.numba_interface.NumbaPlasma
-    estimators: tardis.montecarlo.montecarlo_numba.numba_interface.Estimators
+    r_packet : RPacket
```

Comment:
this should be `tardis.montecarlo.montecarlo_numba.r_packet.RPacket`

Reply:
This also occurs in line 294, should that also be of type tardis.montecarlo.montecarlo_numba.r_packet.RPacket?